bokeh/properties.py

```diff
@@ -337,6 +349,11 @@ def __new__(cls, class_name, bases, class_dict):
         class_dict["__container_props__"] = container_names
         if dataspecs:
             class_dict["_dataspecs"] = dataspecs
+
+        if "__example__" in class_dict:
+            a = class_dict["__example__"]
+            class_dict["__doc__"] += _EXAMPLE_TEMPLATE % (a, a, a)
```

Comment:
This would be much more robust if named placehoders were used, then no need to remember how many times to include the value and you could use more meaningful name than `a`.